kpops/components/base_components/pipeline_component.py

```diff
@@ -250,9 +250,11 @@ async def reset(self, dry_run: bool) -> None:
 
         :param dry_run: Whether to do a dry run of the command
         """
+        await self.destroy(dry_run)
```

Comment:
same question as for the connector, should we really destroy components on reset (by default)?

Reply:
Good point. I moved the logic to the `StreamsBootstrap` class.